WeScanSampleProject/ImplementNewCameraViewController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ImplementNewCameraViewController.swift
+//  WeScanSampleProject
+//
+//  Created by Chawatvish Worrapoj on 7/1/2563 BE.
+//  Copyright © 2563 WeTransfer. All rights reserved.
+//
+
+import UIKit
+import WeScan
+
+class ImplementNewCameraViewController: UIViewController {
```

Comment:
Same as the previous comment

```suggestion
final class NewCameraViewController: UIViewController {
```